src/backends/raster/shapes/bitmap_circle.f90

```diff
@@ -35,20 +37,20 @@ subroutine draw_outer_circle(canva, pixels, circ)
             end if
             x = x + 1
 
-            call draw_pixel(canva, pixels, int(circ%cx) + x, int(circ%cy) + y, stroke_color)
-            call draw_pixel(canva, pixels, int(circ%cx) - x, int(circ%cy) + y, stroke_color)
-            call draw_pixel(canva, pixels, int(circ%cx) + x, int(circ%cy) - y, stroke_color)
-            call draw_pixel(canva, pixels, int(circ%cx) - x, int(circ%cy) - y, stroke_color)
-            call draw_pixel(canva, pixels, int(circ%cx) + y, int(circ%cy) + x, stroke_color)
-            call draw_pixel(canva, pixels, int(circ%cx) - y, int(circ%cy) + x, stroke_color)
-            call draw_pixel(canva, pixels, int(circ%cx) + y, int(circ%cy) - x, stroke_color)
-            call draw_pixel(canva, pixels, int(circ%cx) - y, int(circ%cy) - x, stroke_color)
+            call draw_pixel(canva, pixels, c%x + x, c%y + y, stroke_color)
+            call draw_pixel(canva, pixels, c%x - x, c%y + y, stroke_color)
+            call draw_pixel(canva, pixels, c%x + x, c%y - y, stroke_color)
```

Comment:
It would be a good idea to begin defining overloaded operators for `canvas_point` and `point` derived types, to support at least `+` and `-`